keyboards/annepro2/annepro2.c

```diff
@@ -26,53 +25,29 @@ uint16_t annepro2LedMatrix[MATRIX_ROWS * MATRIX_COLS] = {
 };
 
 void OVERRIDE keyboard_pre_init_kb(void) {
+#if HAL_USE_SPI == TRUE
+    spi_init();
```

Comment:
I'm just curious if this is really related to to the communication. I'm fine with having it in this PR.

Reply:
I missed this from cherry-picking the other PR, I'll take it out since it isn't necessary in this PR